scripts/time-filter-proto.js

```diff
@@ -87,41 +76,9 @@ export default function () {
 }
 
 export function handleSummary(data) {
-    // removing default metrics
-    delete data.metrics['http_req_duration']
-    delete data.metrics['http_req_sending']
-    delete data.metrics['http_req_receiving']
-    delete data.metrics['http_req_blocked']
-    delete data.metrics[`http_req_duration{expected_response:true}`]
-    delete data.metrics['http_req_waiting']
-    delete data.metrics['http_reqs']
-    delete data.metrics['iteration_duration']
-    delete data.metrics['iterations']
-    delete data.metrics['vus']
-    delete data.metrics['http_req_connecting']
-    delete data.metrics['http_req_failed']
-    delete data.metrics['http_req_tls_handshaking']
-
-    data = destinations.reduce((curData, curDestinations) => {
-        return reportPerDestination(curData, curDestinations)
-    }, data)
-
-    return {
-        stdout: textSummary(data, { indent: ' ', enableColors: true }),
-    }
-}
-
-function reportPerDestination(data, destinations) {
```

Comment:
you removed it from proto requests and added only in time-filter. So please, make sure that proto benchmarks still work

Reply:
done, they work